### PR TITLE
Update edlib

### PR DESCRIPTION
**Pull request by karel-brinda**
* [x] I have read the guidelines above.
* [ ] This PR adds a new recipe.
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/edlib/meta.yaml [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
+  version: "1.1.2"
 
 build:
   number: 0
 
 source:
-  fn: v1.0.0.tar.gz
-  url: https://github.com/Martinsos/edlib/archive/v1.0.0.tar.gz
+  fn: v1.1.2.tar.gz
+  url: https://github.com/Martinsos/edlib/archive/v1.1.2.tar.gz
```
Comment (druvus) [inline recipes/edlib/meta.yaml]: Could you please a checksum?

**Comment by karel-brinda** [comment]
Done.

**Comment by karel-brinda** [comment]
@druvus Do you have any idea why I am getting the following error on Travis, please?

```
ERROR bioconda_utils.build:build_recipes(375): BIOCONDA BUILD SUMMARY: of 3 recipes, 1 failed and 0 were skipped. Details of recipes and environments follow.
ERROR bioconda_utils.build:build_recipes(381): BIOCONDA BUILD SUMMARY: while the entire build failed, the following recipes were built successfully:
recipes/metaphlan2
recipes/edlib
ERROR bioconda_utils.build:build_recipes(386): BIOCONDA BUILD SUMMARY: FAILED recipe prodigal-2.60-1.tar.bz2, environment CONDA_GMP=5.1;CONDA_BOOST=1.61;CONDA_PERL=5.22.0;CONDA_NPY=110;CONDA_R=3.3.1;CONDA_HDF5=1.8.17;CONDA_PY=27;CONDA_GSL=1.16;CONDA_NCURSES=5.9
/home/travis/build.sh: line 158:  3507 Terminated              travis_jigger $! $timeout $cmd
```

**Comment by daler** [comment]
@karel-brinda the error is actually here: https://travis-ci.org/bioconda/bioconda-recipes/jobs/203256128#L554. Looks like you need to copy a test fasta (or make one on-the-fly)? 

Some more info on reading the travis log: https://bioconda.github.io/troubleshooting.html#reading-bioconda-utils-logs-on-travis-ci

**Comment by karel-brinda** [comment]
Yes, but this error is caused by a different package which I do not use for edlib (at least I think so). This error appeared after a commit, which was only adding a checksum. Therefore, I think the problem must be somewhere else.

**Comment by karel-brinda** [comment]
I merged the current master and it seems that it solved the problem.

**Pull request merged**